MaterialComponentsBeta.podspec

```diff
@@ -66,6 +66,19 @@ Pod::Spec.new do |mdc|
     extension.dependency "MaterialComponents/schemes/Typography"
   end
 
+  mdc.subspec "BottomNavigation" do |component|
+    component.ios.deployment_target = '8.0'
```

Comment:
Should this be 9.0? I thought some of the APIs were only 9+?

Reply:
The controller is 9+, however when I set the deployment target to 9.0, when installing the pods for the catalog, the installer could not find a compatible version for the unit tests, because the catalog's target deployment version is 8.0.

We are currently using ```NS_CLASS_AVAILABLE_IOS``` to restrict the controller to 9+.

I am a bit unfamiliar with CocoaPods.  Anything I missed or any tips?

Reply:
Hmm, summoning @ianegordon  and @andrewoverton (tomorrow) in case either knows better. Andrew's been working with CocoaPods a lot recently and likely knows better than I do.  Ian knows more about iOS versioning and CocoaPods, I believe.

I'm not sure if we need to specify iOS 9+ for the target or if we think it's sufficient to put it in the headers.

Reply:
If you have gated any use of iOS 9 APIs, I expect this would work as intended.

Please verify on an iOS 8.0 simulator or device.

Reply:
It crashes on an iOS 8.3 simulator as-is. We'll have to find a way to put a guard in the catalog code.